h/routes.py

```diff
@@ -176,8 +176,7 @@ def includeme(config):
     config.add_route('education', '/education/', static=True)
     config.add_route('for-publishers', '/for-publishers/', static=True)
     config.add_route('fund', '/fund/', static=True)
-    config.add_route(
-        'help-center', 'https://hypothesis.zendesk.com/hc/en-us', static=True)
+    config.add_route( 'help-center', '/help/')
```

Comment:
Whoops! Looks like we lost the `static` argument here?

Pyramid docs are always a bit arcane, but if I'm understanding the [documentation](https://docs.pylonsproject.org/projects/pyramid/en/latest/_modules/pyramid/config/routes.html) correctly, `static` makes sense here because `/help/` will never actually be handled; it's just a "constructed URL."